### ISSUE TITLE
Is the project open for UI Design Improvements?

### ISSUE THREAD

**Issue by bhaskarblur** [openMF/self-service-app#2349]
**Summary:**
Hello there, i am looking forward to contributing in this repository and i would like to improve the UI design and usability of the mobile app, is this project open to accepting UI redesigns?
Please tell so that i can start working, i will include mockups, designs before.

Thanks

**Comment by PratyushSingh07** [comment]
I dont think we are looking at the UI revamp just yet . However, we are in the midst of migrating our project from xml to compose view and we appreciate anyone who is ready to contribute in this endeavour

**Comment by bhaskarblur** [comment]
Oh ok, thanks for your reply.

**Comment by ghost** [comment]
@PratyushSingh07 i Wanted to work on this..can you give me an overview how to do this..

**Comment by PratyushSingh07** [comment]
@Thanush66   Currently a bulk of our project is in xml . Hence we want to migrate the project from xml to compose view and while migrating we want to make sure that we maintain a good quality code. To do so you can look at the migration of  `about us`  fragment from xml to compose . You will find a separate package by the name of `about` that will consists of all the related files

**Comment by bhaskarblur** [comment]
Alr, thanks

**Comment by SekhGulamMainuddin** [comment]
Hi, @PratyushSingh07 can I work on the XML to compose migration?

**Comment by PratyushSingh07** [comment]
Ask jawid regarding that

**Comment by SekhGulamMainuddin** [comment]
> Ask jawid regarding that

Ok

Hi @jawidMuhammadi can I work on the XML to compose migration?

**Comment by SatyajitRedekar** [comment]
@PratyushSingh07 I want to work on this issue please assign me this issue !
